### PR TITLE
Add deprecation note

### LINKED ISSUE #1: Run all `make` commands in container.
From https://github.com/gardener/gardener-extensions/issues/122

> All `make` commands require various dev resources to be available:

> - `golang`
> - `helm`
> - various `gnu` utils  such as `tar`, `gzip`, `tr`, `base64`, `cut`, `find`, `grep`, `xargs`.
> - `gingo`
> - linters.

> I think that the most reasonable way to get reproducible builds and things such as `make generate` is to run those commands in a pre-build container image.

### LINKED ISSUE #38: Enhance this controller to support also Alicloud
**How to categorize this issue?**

/area os dev-productivity
/kind enhancement
/priority 3
/os ubuntu

**What would you like to be added**:
Now when this extension is using bash script instead of cloud init, it should be able to bootstrap ubuntu nodes also on Alicloud.
The alicloud version has some [configurations](https://github.com/gardener/gardener-extension-os-ubuntu-alicloud/blob/4e5569c4c3ce5ad40299ca84e1bd68228db090f0/pkg/generator/templates/cloud-init.ubuntu-ali.template#L14-L23) that are missing in this one. They are looking general enough and might be needed also for this extension.

The alicloud version is also [setting](https://github.com/gardener/gardener-extension-os-ubuntu-alicloud/blob/4e5569c4c3ce5ad40299ca84e1bd68228db090f0/pkg/generator/templates/cloud-init.ubuntu-ali.template#L54-L57) `PROVIDER_ID` environment  variable, but since some time it is properly [injected](https://github.com/gardener/gardener-extension-provider-alicloud/blob/a8ccd95d46bf9d51ce331ef0b60fa97f19cb3496/pkg/webhook/controlplane/ensurer.go#L101-L127) by the provider-alicloud and does not need to be migrated.

**Why is this needed**:
To maintain only one os ubuntu extension.

### PR DESCRIPTION
**Pull request by vpnachev**
**What this PR does / why we need it**:
Deprecate this extension in favor of https://github.com/gardener/gardener-extension-os-ubuntu which supports Ubuntu@Alicloud in version v1.11.0.

**Which issue(s) this PR fixes**:
Fixes https://github.com/gardener/gardener-extension-os-ubuntu/issues/38
Closes #1

**Special notes for your reviewer**:
Once this pr is merged, this repo can be archived and moved to https://github.com/gardener-attic

**Release note**:

```improvement operator

```

### CODE REVIEW

**Comment by vpnachev** [comment]
/invite @gardener/gardener-extension-provider-alicloud-maintainers 
/invite  @gardener/gardener-extension-os-ubuntu-maintainers

File: README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 [![CI Build status](https://concourse.ci.gardener.cloud/api/v1/teams/gardener/pipelines/gardener-extension-os-ubuntu-alicloud-master/jobs/master-head-update-job/badge)](https://concourse.ci.gardener.cloud/teams/gardener/pipelines/gardener-extension-os-ubuntu-alicloud-master/jobs/master-head-update-job)
 [![Go Report Card](https://goreportcard.com/badge/github.com/gardener/gardener-extension-os-ubuntu-alicloud)](https://goreportcard.com/report/github.com/gardener/gardener-extension-os-ubuntu-alicloud)
 
+# Deprecation Note
```
Comment (ialidzhikov) [inline README.md]:
```suggestion
## :warning: Deprecation Note
```

File: README.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
...
+
+This repository is not actively maintained anymore in favor of its substitute https://github.com/gardener/gardener-extension-os-ubuntu which supports Ubuntu on Alicloud since version v1.11.0.
+
+# How does it work
```
Comment (ialidzhikov) [inline README.md]:
```suggestion
## How does it work
```

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**